CyberArcCTF-24-Writeups/Web Exploitation/John Will Triumph/Writeup.md

```diff
@@ -0,0 +1,154 @@
+# John Will Triumph
+
+**Author: Harsh Iyer**
+**Points: 300**
+**Difficulty: Medium**
+
```

Comment:
Rename file to README.md and fix formatting
https://github.com/Laav10/CTF-Writeups/blob/368c5064f95199fb9251c4aec8087f934addd3a9/CyberArcCTF-24-Writeups/Web%20Exploitation/John%20Will%20Triumph/Writeup.md#john-will-triumph
(Challenge details are displayed on the same line)

Reply:
ok shall fix it